test/server/models/rating.test.js

```diff
@@ -0,0 +1,32 @@
+import chai from 'chai';
+import models from '../../../server/models';
+import {
+  ratingWithValidData,
+  ratingWithInvalidData
+} from '../../../mockdata/ratingMockData';
+
+const { Rating } = models;
+chai.should();
+
+describe('Test rating model - Unit test', () => {
+  before((done) => {
+    Rating.destroy({ where: {} });
```

Comment:
Great. But remember we agreed not to use this destroy function in our tests because it drops a table that someone else may want to use for a test. Use seeders and test the returned data using findOne or whatever is useful to your testing situation.

Reply:
Thanks, Chris, I have fixed this.